### ISSUE TITLE
Cannot bind "Toggle Guake visibility" to any key in Ubuntu 16.04

### ISSUE THREAD

**Issue by brmc** [Guake/guake#759]
I've read through the issues regarding this bug, but I haven't seen any updates in a while.  Are there any potential work-arounds or is there a version that is known to work?

**Comment by Stibbons** [comment]
it works here (ubuntu 16.04 as well)

**Comment by PombaM** [comment]
Works well for me too (Ubuntu 16.04)

**Comment by PombaM** [comment]
> it works here (ubuntu 16.04 as well)

Ditto

**Comment by ElijahLynn** [comment]
#697

**Comment by ElijahLynn** [comment]
Just completely wiped Guake from my Ubuntu 14.04. system and reinstalled 0.8.7 from https://launchpad.net/~webupd8team/+archive/ubuntu/unstable and still cannot get a keyboard shortcut working. Then I removed with apt-get and cloned repo and did a ./dev.sh --install and same thing.

**Comment by Stibbons** [comment]
Look like the library "keybinder" has a problem on your system. Can you launch guake from another console and see in the output log if there is an error related to keybinder ?

**Comment by ElijahLynn** [comment]
Yeah, I am coming to the same conclusion. However, last night I was playing with it again and looking at the output from ./dev.sh --install and --reinstall. It was never making the virtual environment so I did see that I needed to `sudo apt-get install python-virtualenv`. That did allow it to install in a virtual environment but still doesn't work and when it launches it says launch with ctrl+alt+t but that doesn't work and I don't know where it is getting that from as I never set that and they are virtualenv conf settings, not inside the gconf settings. I tried uninstalling python-keybinder and libkeybinder0 but no luck so far. I feel like there must be some conflicting package somewhere doing something funky.

**Comment by ElijahLynn** [comment]
Okay, did some more debugging and downloaded the sample at https://raw.githubusercontent.com/engla/keybinder/master/examples/example.py and it worked with the <Ctrl>A shortcut by default. I changed it to <Shift><Alt>T and it stopped working so it appears something is funky with my keybinder.
